### PR TITLE
patch: project creation

### PR DESCRIPTION
**Pull request by swand0g**
### major changes 👉
- updates form to change spaces to dashes when creating new projects

https://user-images.githubusercontent.com/22817788/171926747-119be01f-f027-4f6f-997e-077475ea912f.mov

### CODE REVIEW

File: src/components/projects/new-project.jsx [hunk 1]
```diff
@@ -45,6 +52,11 @@ export default function NewProject({ isOpen, onOpen, onClose }) {
     }
   }
 
+  const validateProjectName = (value) => {
```
Comment (eric-hc) [inline src/components/projects/new-project.jsx:55]:
```suggestion
  const validateProjectName = value => {
```
my opinion is an opinion

File: src/components/projects/new-project.jsx [hunk 2]
```diff
@@ -57,8 +69,15 @@ export default function NewProject({ isOpen, onOpen, onClose }) {
             {(props) => (
               <Form>
                 <Stack spacing='2'>
-                  <FormikField name="name" label="Project name" validation={stringIsNotEmpty} />
+                  <Tooltip label>
```
Comment (eric-hc) [inline src/components/projects/new-project.jsx:72]: what does `label` do here? shouldnt it be set to something?

**Review by eric-hc — CHANGES_REQUESTED** [review]

File: src/components/projects/new-project.jsx (hunk 2 above)
Comment (swand0g) [reply src/components/projects/new-project.jsx:72]: omg I did not mean to push this

**Review by eric-hc — APPROVED** [review]

File: src/components/projects/new-project.jsx [hunk 3]
```diff
@@ -1,32 +1,39 @@
...
 
 export default function NewProject({ isOpen, onOpen, onClose }) {
   const toast = useToast()
   const { mutate } = useSWRConfig()
   const { currentTeam } = useContext(TeamContext)
+  const [projectNameValue, setProjectNameValue] = useState('')
+  const shouldShowDashesText = projectNameValue.trim().includes(' ')
   
   const onSubmit = (values, { setSubmitting }) => {
+    const formattedProjectName = values.name.trim().replaceAll(' ', '-')
```
Comment (tiffkwin) [inline src/components/projects/new-project.jsx:21]: can this use projectNameValue instead?

**Review by tiffkwin — CHANGES_REQUESTED** [review]

File: src/components/projects/new-project.jsx [hunk 4]
```diff
@@ -39,12 +46,17 @@ export default function NewProject({ isOpen, onOpen, onClose }) {
...
 
-  const stringIsNotEmpty = (value) => {
+  const stringIsNotEmpty = value => {
     if (!value?.length > 0) {
       return 'Value must not be empty!'
     }
   }
 
+  const validateProjectName = value => {
+    setProjectNameValue(value)
```
Comment (tiffkwin) [inline src/components/projects/new-project.jsx:56]: can you set this to be value.trim() and reuse projectNameValue so you don't have to call trim() repeatedly

**Review by tiffkwin — CHANGES_REQUESTED** [review]

**Review by tiffkwin — APPROVED** [review]

**Pull request merged**